dbt_coves/utils/flags.py

```diff
@@ -101,3 +106,23 @@ def parse_args(self, cli_args: List[str] = list()) -> None:
             if self.task == "check":
                 if self.args.no_fix:
                     self.check["no-fix"] = self.args.no_fix
+
+            if self.task == "airbyte" and self.args.cls.__name__ == "LoadAirbyteTask":
+                if self.args.path:
+                    self.load["airbyte"]["path"] = self.args.path
+
+                if self.args.host and self.args.port:
+                    self.load["airbyte"]["port"] = self.args.port
+                    self.load["airbyte"]["host"] = self.args.host
+                if self.args.secrets:
+                    self.load["airbyte"]["secrets"] = self.args.secrets
+
+            if (
+                self.task == "airbyte"
```

Comment:
You can remove `self.task == "airbyte" and `

Reply:
e6216cc[](url)